src/pages/Contact.tsx

```diff
@@ -2,10 +2,55 @@ import * as React from "react";
 
 import Container from "../components/Container";
 import PageHeading from "../components/PageHeading";
+import styled from "../styled-components";
+import {
+  TwitterTimelineEmbed,
+  TwitterFollowButton
+} from "react-twitter-embed/dist/index.js";
+
+const TwitterBox = styled.div`
+  max-width: 400px;
+  max-height: 600px;
+  margin: 2em;
+`;
+
+const ContactInfoBox = styled.section`
+  display: flex;
+  flex-direction: row;
+  justify-content: space-around;
+  margin: 2em 0;
+
+  & > * {
+    flex-basis: 0;
+    flex-grow: 1;
+    margin: 0 2em;
+  }
+
+  & > *:last-child {
+    margin-top: 1em;
+  }
+`;
 
 const Contact = () => (
   <Container>
     <PageHeading>Contact</PageHeading>
+    <ContactInfoBox>
+      <div>
+        <h3>Darkside 2018 Contact</h3>
+        Jesse Kovacs
+        <br /> jjk007@live.unc.edu
+      </div>
+      <div>
+        <TwitterFollowButton screenName={"UNC_Darkside"} />
```

Comment:
It doesn't really matter, but you don't need the curly braces for a literal string.

```suggestion
         <TwitterFollowButton screenName="UNC_Darkside" />
```